tests/test_monkeypatch.py

```diff
@@ -33,3 +37,46 @@ def test_view_published_in_toolbar_in_preview_mode_button_url(self):
         self.assertContains(
             response,  'class="cms-btn cms-btn cms-btn-switch-save" target="_blank" >View Published</a>'
         )
+
+
+class VersioningIntegrationTestCase(CMSTestCase):
+    def test_versioning_changelist_published_date(self):
+        """
+        Monkey patch should add expiry column and values to admin menu list display
+        """
+        published_version = factories.PageVersionFactory(content__template="page.html", content__language="en")
+        version_admin = admin.site._registry[published_version.versionable.version_model_proxy]
+
+        request = RequestFactory().get("/")
+        list_display = version_admin.get_list_display(request)
+
+        # Published date field should have been added by the monkeypatch
+        self.assertIn('published_date', list_display)
+        self.assertNotIn('created', list_display)
+        self.assertEqual(PAGEADMIN_PUBLISHED_DATE_FIELD_LABEL, version_admin.published_date.short_description)
+
+    def test_when_created_not_in_list_display(self):
+        """
+        Monkey patch should return the default list display when created is available
+        """
+        def mock_get_list_display(self, request):
```

Comment:
I thought of another way you could have done this, by creating mock object to pass to your function:
```
mock_get_list_display = MagicMock(spec=VersionAdmin, return_value=["foo", "bar"])
```
But current implementation is fine, just adding for reference